app/src/main/java/com/vci/vectorcamapp/intake/presentation/IntakeScreen.kt

```diff
@@ -118,6 +125,43 @@ fun IntakeScreen(
                     )
                 }
 
+                Tooltip(
+                    isVisible = state.isCollectionMethodTooltipVisible,
+                    onClick = { onAction(IntakeAction.ShowCollectionMethodTooltipDialog) },
+                    onDismiss = { onAction(IntakeAction.HideCollectionMethodTooltipDialog) },
+                    buttonText = "Tap to learn more about collection methods"
+                ) {
+                    Column(
+                        verticalArrangement = Arrangement.spacedBy(MaterialTheme.dimensions.spacingMedium)
+                    ) {
+                        Text(
+                            text = "Collection Methods",
+                            style = MaterialTheme.typography.titleLarge,
+                            color = MaterialTheme.colors.textPrimary,
+                            fontWeight = FontWeight.Bold,
+                            modifier = Modifier.padding(bottom = MaterialTheme.dimensions.paddingSmall)
+                        )
+                        CollectionMethodTooltipRow(
+                            title = "CDC Light Trap",
+                            description = "A light trap that uses a battery to attract and collect mosquitoes during the night.",
+                            iconPainter = painterResource(id = R.drawable.ic_light),
```

Comment:
I would replace this with a picture of a light trap icon. I think canva has some icons you can use. Let me know, and I can share an SVG potentially.